frontend/src/js/controllers/authCtrl.js

```diff
@@ -55,26 +55,30 @@
             vm.wrnMsg = {};
         }
 
-        vm.passwordChecksignUp = function(password1, password2) {
+        vm.passwordChecks = function(password1, password2) {
             var password1_len = password1.length;
             var password2_len = password2.length;
+            var context = {};
 
             if (password1_len >= 8 && password2_len >= 8)
             {
                 if (password1 === password2)
                 {
-                    vm.confirmMsg = "Passwords Match !"
-                    return true;
+                    context.confirmMsg = "Passwords Match !";
+                    context.status = true;
+                    return context;
                 }
                 else {
-                    vm.confirmMsg = "Passwords do not Match !"
+                    context.confirmMsg = "Passwords do not Match !";
+                    context.status = false;
                     vm.stopLoader();
```

Comment:
No need to stop loader here. remove this line since this event is already handled at #L155